src/typing/typeloadFields.ml

```diff
@@ -917,7 +917,7 @@ module TypeBinding = struct
 						e
 					end in
 					e
-				| Var v when v.v_read = AccInline ->
+				| Var v when v.v_read = AccInline && not (defined ctx.com Define.NoInline) ->
```

Comment:
I think we might want to use `ctx.doinline` instead because that also respects the display mode setting.